### PR TITLE
Add la netinha

### PR DESCRIPTION
**Pull request by neto3000**

### CODE REVIEW

File: scripts/netinha.coffee [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = (reylero) ->
+ reylero.hear /netinha/i, (res) ->
+   res.send "Soooooo, You are you gonna have the stand-up from home, right?"
+
+ reylero.respond /la netinha/i, (res) ->
+   res.reply "Maigos I will be having the stand-up from home, I'll be going to the office after."
+
+  reylero.respond /no today/i, (res) ->
```
Comment (FerPerales) [inline scripts/netinha.coffee:8]: Check identation :)

Comment (joiggama) [inline scripts/netinha.coffee:8]: :+1: to this, this indentation will break the build process.

**Comment by joiggama** [comment]
@neto3000 I also have an open PR, I'm adding tests for it later this week, let's do the same for your PR. I was thinking about using [mocha](https://mochajs.org) and/or [chai](http://chaijs.com/).

File: scripts/netinha.coffee [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module.exports = (reylero) ->
+ reylero.hear /netinha/i, (res) ->
+   res.send "Soooooo, You are you gonna have the stand-up from home, right?"
+
+ reylero.respond /la netinha/i, (res) ->
+   res.reply "Maigos I will be having the stand-up from home, I'll be going to the office after."
```
Comment (isay-sosa) [inline scripts/netinha.coffee:6]: `I'll be at the office later on` maybe?

**Comment by joiggama** [comment]
No updates in a year!

**Pull request closed without merging**